bundles/AdminBundle/Resources/public/js/pimcore/object/tags/multihrefMetadata.js

```diff
@@ -114,6 +114,10 @@ pimcore.object.tags.multihrefMetadata = Class.create(pimcore.object.tags.abstrac
                 cellEditor = function() {
                     return new Ext.form.TextField({});
                 };
+            } else if (this.fieldConfig.columns[i].type == "date" && !readOnly) {
+                cellEditor = function() {
+                    return new Ext.form.DateField({});
```

Comment:
Imho we should configure a `submitFormat` here, to ensure the format is always the same.